lang/golang/golang/Makefile

```diff
@@ -202,17 +213,21 @@ endif
 $(eval $(call GoCompiler/AddProfile,Host,$(HOST_BUILD_DIR),$(HOST_GO_PREFIX),$(HOST_GO_VERSION_ID),$(GO_HOST_OS_ARCH),$(HOST_GO_INSTALL_SUFFIX)))
 
 HOST_GO_VARS= \
+	GOHOSTOS="$(GO_HOST_OS)" \
+	GOHOSTARCH="$(GO_HOST_ARCH)" \
```

Comment:
Normally `GOHOSTARCH` (and `GOHOSTOS`) are auto-detected by go itself, so there should be no need to set it. What error messages are you getting?

Reply:
it is autodetected as "arm64" but I am using external bootstrap from a Go install that is "arm" and build fails

I'll post the error soon

Reply:
in the meantime I can try dropping GOHOSTOS= and see if it still works